.github/workflows/test.yml

```diff
@@ -53,8 +53,8 @@ jobs:
 
     steps:
       - name: "Setup postgres config"
+        # docker exec -i ${{ job.services.postgres.id }} apt update
```

Comment:
What's this?

Reply:
We used that to install wal2json.

Reply:
All the tests pass in GH Actions w/o this line. I'll remove the comment

Reply:
We removed the step to install wal2json in Jan when we updated our GH actions to use PG 16: https://github.com/dbos-inc/dbos-transact/pull/254